src/utils/test-utils.js

```diff
@@ -9,3 +11,28 @@ export async function renderAndWait (ui, ms = 0, options = {}) {
   })
   return queries
 }
+
+export const renderAndWaitWithRouter = async (ui, ms = 0, options = {}) => {
```

Comment:
so my preference here would be that we just combine them in the code, and export them separately. There's something about this being one monolithic function that is unwieldy. Like, how do we then combine it with the modal code that you have in the other pr? We end up with `renderAndWaitWithRouterAndModal` and etc etc.

Instead if we just export `wrapInRouter` and `renderAndWait` we can compose them in the test

`renderAndWait(wrapInRouter(<Component />))`